docs/docs/domain.mdx

```diff
@@ -384,6 +384,36 @@ class NumberOfPeopleSlot(Slot):
         return r
 ```
 
+A custom slot class is implemented as an independent python module, 
```

Comment:
The content is good, but can you please use active voice? It'd be great if you could also update the text above to active voice. (See [tone and language guildelines](https://www.notion.so/rasa/Tone-and-language-guidelines-cd96f21292b24c1ab7f7457c7973ec69).)